config/config.exs

```diff
@@ -84,4 +84,13 @@ config :sentry,
   root_source_code_paths: [File.cwd!()],
   context_lines: 5
 
+
+config :dotcom, Dotcom.Scheduler,
+  jobs: [
+    commuter_rail_status: [
+      schedule: {:extended, "*/30"}, # Runs every two seconds
+      task: {Dotcom.SystemStatus.CommuterRail, :commuter_rail_status, []}
```

Comment:
I think it might be simpler to have this trigger a new function which _computes and broadcasts_  the commuter rail status (perhaps with the last updated time) every 5 or whatever minutes (and revert `commuter_rail_status/0` to the simpler, uncached version which doesn't itself handle broadcasting)